webapp/app/routes/app/proposals/$id/proposal-state.tsx

```diff
@@ -0,0 +1,50 @@
+import { PROPOSAL_STATES, type StatusTime } from "@/utils/proposal-states";
+
+export type ProposalStateProps = {
+  status: PROPOSAL_STATES;
+  times: StatusTime | null;
+};
+
+export default function ProposalState({ status, times }: ProposalStateProps) {
```

Comment:
Include comment with link to the [schedule](https://docs.zknation.io/zksync-governance/schedule-1-standard-governance-procedures) where presumably you got this info from

Reply:
I actually took this from the contract, not the docs. But I feel that linking the docs is better. Adding this.